Mathlib/Topology/PartialHomeomorph.lean

```diff
@@ -1041,8 +1041,8 @@ section Prod
 /-- The product of two partial homeomorphisms, as a partial homeomorphism on the product space. -/
 @[simps! (config := mfld_cfg) toPartialEquiv apply,
   simps! (config := .lemmasOnly) source target symm_apply]
-def prod (e : PartialHomeomorph α β) (e' : PartialHomeomorph γ δ) :
-    PartialHomeomorph (α × γ) (β × δ) where
+def prod (e : PartialHomeomorph X Y) (e' : PartialHomeomorph Z Z') :
```

Comment:
I would use something like `(eX : PartialHomeomorph X X') (eY : PartialHomeomorph Y Y')`, similarly to [Homeomorph.prodCongr](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Homeomorph.html#Homeomorph.prodCongr).

Reply:
That's much better; switched to it.